apps/cookbook/src/app/examples/modal-example/alert-example/modal-example-embedded-alert.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, OnInit, Optional, SkipSelf } from '@angular/core';
+import { AlertConfig, Modal } from '@kirbydesign/designsystem';
+
+const config = {
+  selector: 'cookbook-modal-embedded-alert-example',
+  template: `
+    <kirby-page-title>Modal with alert</kirby-page-title>
+    <p>
+      <em>
+        This modal will ask the user if they are sure they want to close the modal if they have
+        entered data in any of the form fields.
+      </em>
+    </p>
```

Comment:
Consider styling this paragraph to stand out more - e.g. by using our flag component or similar and/or adding a "Please note" in bold...